### PR TITLE
added wren recipe

### PR DESCRIPTION
**Pull request by DerThorsten**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by DerThorsten** [comment]
not sure if a smth like  arch=generic is missing

**Comment by DerThorsten** [comment]
...looks like the win build failed silent

**Comment by DerThorsten** [comment]
Since I cannot make a win build with provided *.sol file (https://github.com/wren-lang/wren/tree/main/projects/vs2017) I provided a minimal CMakeLists.txt in the recipe folder.
It would be nicer to have this https://github.com/wren-lang/wren/pull/997  merged in wren, but there seems to be reservations about using CMake instead of premake

File: recipes/wren/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - cmake
+    - make  # [unix]
+  host:
+  run:
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/wren.h                                                      # [unix]
+    - test -f ${PREFIX}/include/wren.hpp                                                    # [unix]
+    - test -f ${PREFIX}/lib/libwren.a                                                       # [unix]
```
Comment (wolfv) [inline recipes/wren/meta.yaml:36]: static libs are forbidden :) see CFEP 13 or something like that

File: recipes/wren/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Wren is a small, fast, class-based concurrent scripting language.'
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    Wren is a small, fast, class-based concurrent scripting language.
+    Think Smalltalk in a Lua-sized package with a dash of Erlang 
+    and wrapped up in a familiar, modern syntax.
+  doc_url: https://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```
Comment (wolfv) [inline recipes/wren/meta.yaml:52]: this looks wrong

File: recipes/wren/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "wren" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
```
Comment (wolfv) [inline recipes/wren/meta.yaml:14]: empty line looks ugly

File: recipes/wren/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
```
Comment (wolfv) [inline recipes/wren/meta.yaml:4]: comments shoudl be removed

**Comment by wolfv** [comment]
Looks great except for the comments I left! :tada:

File: recipes/wren/build.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+cmake -DCMAKE_BUILD_TYPE=Release      \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX  \
+      -DCMAKE_PREFIX_PATH=$PREFIX     \
+      -DCMAKE_INSTALL_LIBDIR=lib      \
+      -DSRC_DIR=${SRC_DIR} \
+      ${RECIPE_DIR}
+
+make -j2 install
```
Comment (wolfv) [inline recipes/wren/build.sh:8]: `-j${CPU_COUNT}` is better as that will scale to whatever local machine you have etc.

**Comment by wolfv** [comment]
shoot there is one more minor thing :'(

**Pull request merged**